yast2_rmt-config_written_successfully-20190124.json

```diff
@@ -0,0 +1,22 @@
+{
+  "area": [
+    {
+      "xpos": 370,
+      "ypos": 350,
+      "width": 166,
+      "height": 20,
+      "type": "match"
+    },
+    {
+      "xpos": 383,
+      "ypos": 320,
+      "width": 66,
+      "height": 16,
+      "type": "match"
+    }
+  ],
+  "properties": [],
+  "tags": [
+    "config_written_successfully"

```

Comment:
that tag sounds a bit too generic. Please use a prefix as in the next needle

Reply:
Right, I missed that one.

Reply:
Done